benchmark.php

```diff
@@ -43,12 +49,8 @@
         break;
 }
 
-
 $time = microtime(true) - $startTime;
 echo "\n";
 echo "method      : $method\n";
 echo "elem count  : $ct\n";
 echo "time passed : $time s\n";
-
-// Useless since the memory used by libxml libraries is not reported
```

Comment:
I even think this file is not relevant and could be removed. 
And just for info, if you get back to php, there is an awesome benchmarking tool called PhpBench to do this kind of benchmark.